pages.de/common/kubectl-get.md

```diff
@@ -0,0 +1,32 @@
+# kubectl get
+
+> Abfragen von Kubernetes Resourcen und Objekten.
+> Weitere Informationen: <https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#get>.
+
+- Alle Namespaces im aktuellen Cluster anzeigen:
+
+`kubectl get namespaces`
+
+- Alle Nodes in spezifischen Namepsace abfragen:
```

Comment:
I think this sounds a little better
```suggestion
- Alle Nodes in einem bestimmten Namepsace abfragen:
```
Same for all the other examples in this page.